lib/MarcVersionHistory/utils.js

```diff
@@ -0,0 +1,124 @@
+import { Link } from 'react-router-dom';
+import sortBy from 'lodash/sortBy';
+
+import { formatDateTime } from '@folio/stripes-acq-components';
+
+const prefixFieldName = (field, formatMessage) => formatMessage(
+  { id: 'stripes-marc-components.versionHistory.fieldPrefix' },
+  { field },
+);
+
+export const transformVersion = (version, formatMessage) => {
+  const { diff: marcDiff, eventDate } = version;
+  const diff = {
+    fieldChanges: [],
+  };
+
+  // Process added fields
+  if (marcDiff.added) {
+    marcDiff.added.forEach(entry => {
+      diff.fieldChanges.push({
+        changeType: 'ADDED',
+        fieldName: prefixFieldName(entry.field, formatMessage),
+        newValue: entry.newValue,
+      });
+    });
+  }
+
+  // Process modified fields
+  if (marcDiff.modified) {
+    marcDiff.modified.forEach(entry => {
+      diff.fieldChanges.push({
+        changeType: 'MODIFIED',

```

Comment:
Let's create a constant for change types and apply it in `getActionLabel` as well.